paddle/framework/operator.h

```diff
@@ -290,6 +290,15 @@ class ExecutionContext {
     return device_context_;
   }
 
+  //! Get a input which has multiple variables.
```

Comment:
the comment is not accurate, all our input/output is stored in a vector.

Reply:
Done.